src/iterative_ensemble_smoother/sies.py

```diff
@@ -17,50 +17,61 @@
 
 
 class SIES:
-    """
-    Initialize a Subspace Iterative Ensemble Smoother (SIES) instance.
+    r"""
+    Implement a Sequential Iterative Ensemble Smoother (SIES) for Data Assimilation.
+
+    This is an implementation of the algorithm described in the paper: "Efficient
+    Implementation of an Iterative Ensemble Smoother for Data Assimilation
+    and Reservoir History Matching", written by :cite:t:`evensen2019efficient`.
 
-    This is an implementation of the algorithm described in the paper:
-    Efficient Implementation of an Iterative Ensemble Smoother for
-    Data Assimilation and Reservoir History Matching
-    written by Evensen et al (2019),
-    URL: https://www.frontiersin.org/articles/10.3389/fams.2019.00047/full
+    Note
+    ----
+    The class attributes follow the paper nomenclature.
 
-    Parameters
+    Attributes
```

Comment:
I think we should avoid "exposing" attributes in the docs. They are meant for internal use by the class and could in principle change names. Users should interact via the API, not via attributes (unlike e.g. sklearn models, where users are also encouraged to get e.g. `coefs_` from a linear model).

At least that is my thinking. I think exposing attributes leads to more noise, more stuff to maintain, more backwards compatibility issues, etc.

Reply:
Ok I removed it. And you are right, this is anyway  to discuss in another PR anyway.